src/main/java/core/basesyntax/io/read/ReportReader.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.io.read;
+
+import java.io.FileNotFoundException;
+
+public interface ReportReader {
+    public String readReport() throws FileNotFoundException;
```

Comment:
The method signature in the interface should not declare that it throws a specific exception like FileNotFoundException. It's better to keep the interface method signatures clean and handle exceptions in the implementation classes.